actions/changelog-generate/action.yaml

```diff
@@ -50,24 +58,30 @@ runs:
         fi
       shell: bash
     - name: Generate latest only changelog
-      uses: orhun/git-cliff-action@v4
+      # there is no proper release for the install dir fix, yet
+      uses: orhun/git-cliff-action@104a6cf3c9aa0fdfe4eab129f9c1900e1eb8f7fd
```

Comment:
This was released now in https://github.com/orhun/git-cliff-action/releases/tag/v4.5.0 and is therefore also part of `v4`.

Same for the `git-cliff-action` usage below.